.github/actions/next-stats-action/Dockerfile

```diff
@@ -7,7 +7,8 @@ LABEL repository="https://github.com/vercel/next-stats-action"
 COPY . /next-stats
 
 # Install node_modules
-RUN cd /next-stats && yarn install --production
+RUN npm i -g pnpm
```

Comment:
Should this use the same version as `PNPM_VERSION` below?

Reply:
Yeah we can lock it here as well